internal/escape/escape.go

```diff
@@ -138,3 +153,54 @@ func Unescape(s string) string {
 	}
 	return string(unescaped)
 }
+
+// URLEscape uses url.PathEscape to escape s.
+func URLEscape(s string) string {
```

Comment:
Since you now have URLEscape, how about renaming Escape to HexEscape?

Reply:
Done.